.devcontainer/devcontainer.json

```diff
@@ -1,5 +1,9 @@
 {
-	"image": "mcr.microsoft.com/devcontainers/javascript-node:0-18",
+	"image": "mcr.microsoft.com/devcontainers/javascript-node:1-18-bookworm",
```

Comment:
This won't play nice with `docker-in-docker` yet :(

Reply:
I thought so too, but it strangely worked 😅 

I tested with few things -
- dev container was created fine
- docker works
- devcontainer/cli works

@joshspicer  Should I check something else?

But anyhow, let's switch to `bullseye` to be safe. The current dev container will still point to `bookworm`

Reply:
Updated in [Update devcontainer.json](https://github.com/devcontainers/templates/pull/172/commits/85d40d9aedc2e5661b83ea52978041f81574b421)